Makefile

```diff
@@ -23,6 +23,11 @@ package:
 
 upload:
 	poetry publish --verbose --username ${PYPI_USERNAME} --password ${PYPI_PASSWORD}
+
+upload-spruce:
```

Comment:
poetry requires a configuration command to set up the repository for test-pypi: https://python-poetry.org/docs/repositories/#publishable-repositories